packages/sample/src/main.ts

```diff
@@ -17,7 +26,27 @@ program
     .command("led")
     .description("Run LED steps")
     .action(async () => {
-        await led();
+        let hubs = await getExpansionHubs();
+        await led(hubs);
     });
 
 program.parse(process.argv);
+
+async function getExpansionHubs(): Promise<ExpansionHub[]> {
+    let options = program.opts();
+    let serialNumber = options.serialNumber;
+    let moduleAddress = Number(options.address);
```

Comment:
If the address was not specified, this will be `NaN`, which is not ideal. Instead, I'd type this as `number | undefined`, and set it to `undefined` if it's `NaN`.